webapp/src/services/api.ts

```diff
@@ -18,8 +18,12 @@ const responseToData =
     try {
       const response = await responsePromise(arg);
       return { data: await response.json() };
-    } catch {
-      return { error: { message } };
+    } catch (err: unknown) {
```

Comment:
This `err` is already `unknown`, no need to specify it.
```suggestion
    } catch (err) {
```